### PR TITLE
Settings

### PR DESCRIPTION
**Pull request by MauriceHendrix**
- Trying to clarify Readme
- Added settings for running in a sub folder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,15 @@ git clone git@github.com:ModellingWebLab/deployment.git
 cd deployment
 git submodule update --init
 ```
+### Running in a folder
+If you're running weblab in a folder e.g. http://myhost.com/weblab/ then edit `roles/django/templates/deployed.j2` and replace '/' with the correct folder e.g.:
+
+```
+FORCE_SCRIPT_NAME = '/weblab/'
+STATIC_URL =+ '/static/weblab/'
```
Comment (jonc125) [inline README.md:24]: Shouldn't this be `/weblab/static/`?

File: roles/django/templates/deployed.j2 [hunk 2]
```diff
@@ -6,6 +6,11 @@ from pathlib import Path
 
 from .base import *  # noqa
 
+FORCE_SCRIPT_NAME = '/'
+STATIC_URL =+ '/static/'
```
Comment (jonc125) [inline roles/django/templates/deployed.j2:10]: Did you really mean `=+` ???

File: roles/django/templates/deployed.j2 [hunk 3]
```diff
@@ -6,6 +6,11 @@ from pathlib import Path
 
 from .base import *  # noqa
 
+FORCE_SCRIPT_NAME = '/'
```
Comment (jonc125) [inline roles/django/templates/deployed.j2:9]: Given that this worked before without these overrides, I'd be tempted instead to revert these changes, and just say in the readme that suitable lines need to be *added*.

**Review by jonc125 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MauriceHendrix) [reply README.md:24]: you're quite right of ocurse

File: roles/django/templates/deployed.j2 (hunk 2 above)
Comment (MauriceHendrix) [reply roles/django/templates/deployed.j2:10]: uups

File: roles/django/templates/deployed.j2 (hunk 3 above)
Comment (MauriceHendrix) [reply roles/django/templates/deployed.j2]: yeah ok

File: README.md [hunk 4]
```diff
@@ -16,6 +16,15 @@ git clone git@github.com:ModellingWebLab/deployment.git
 cd deployment
 git submodule update --init
 ```
+### Running in a folder
+If you're running weblab in a folder e.g. http://myhost.com/weblab/ then edit `roles/django/templates/deployed.j2` and add the following lines with the correct folder name e.g.:
```
Comment (jonc125) [inline README.md:20]:
```suggestion
### Running at a sub-URL

If you're hosting WebLab at a sub-URL on a domain (e.g. http://myhost.com/weblab/ rather than just http://myhost.com/) then edit `roles/django/templates/deployed.j2` and add the following lines with the correct URL prefix name, e.g.:
```

File: README.md [hunk 5]
```diff
@@ -59,6 +68,7 @@ If this fails, make appropriate changes (and update these instructions!), and ru
 ```shell
 vagrant up --provision
 ```
+#### Add initial admin/superuser
```
Comment (jonc125) [inline README.md:71]:
```suggestion

#### Add initial admin/superuser
```

File: README.md [hunk 6]
```diff
@@ -16,6 +16,15 @@ git clone git@github.com:ModellingWebLab/deployment.git
...
 ```
+### Running in a folder
+If you're running weblab in a folder e.g. http://myhost.com/weblab/ then edit `roles/django/templates/deployed.j2` and add the following lines with the correct folder name e.g.:
+
+```
+FORCE_SCRIPT_NAME = '/weblab/'
+STATIC_URL = '/weblab/static/'
+LOGIN_REDIRECT_URL = '/weblab/'
+LOGOUT_REDIRECT_URL = '/weblab/'
+```
```
Comment (jonc125) [inline README.md:28]:
```suggestion

Note that you will need to set up the appropriate proxying *outside* the Web Lab deployment so that requests to your sub-URL come through to the nginx server used by Web Lab. Such configuration is outside the scope of this guide.

```

**Review by jonc125 — APPROVED** [review]
A few wording improvements but otherwise OK I think.
